app/js/models/captain.js

```diff
@@ -4,8 +4,16 @@ define(
         var Model = Backbone.Model.extend({
             // matches first part of method name in @remote.method
             urlRoot: '/wsgi_service.captain_',
-            defaults: {
-                tshirt_size: 'Large'
+            validate: function(attrs){
+                if(!attrs.name){
+                    this.errorModel.set({name: "Please enter captain name."})
+                }
+                if (!attrs.email){
```

Comment:
Luke, Even though email is not required in ndb_models, I get a server error if email not entered into form.
Is this the best way to handle this?

Reply:
Yes, it's best to catch any errors within the browser layer, if you can.  Because you can give quick feedback to the user. 

The email field should not be required.  There are some captains who don't use email.  We should fix the server error.  Can you file a separate issue please?